build_library/generate_au_zip.py

```diff
@@ -335,6 +343,8 @@ def main():
                     default='au-generator.zip', help='Name of the zip file')
   parser.add_option('-k', '--keep-temp', dest='keep_temp', default=False,
                     action='store_true', help='Keep the temp files...',)
+  parser.add_option('-a', '--arch', dest='arch',
+                     default='amd64', help='Arch Default: amd64',)
```

Comment:
```suggestion
                     default='amd64', help='Arch amd64/arm64. Default: amd64',)
```
Maybe optparse can do choices? Anyway it helps the user to know the possible values.

Reply:
Changed so